src/events/message-handler.ts

```diff
@@ -1,17 +0,0 @@
-import { Message } from 'discord.js';
```

Comment:
If you do decide to delete this, you might want to remove the normal message intents too:
![image](https://user-images.githubusercontent.com/11231402/187571644-341299b4-d517-4df7-88f1-21794d41d0dc.png)

Reply:
If it does affect caching wouldn't that be behind the scenes? I am simply removing the code which handles the events, I will still get the events

Reply:
Yes, if you keep the intents you will still get the events.

Reply:
So do you think I should or shouldn't keep it? If it affects caching I would think I wouldn't want to remove it

Reply:
I would leave it then to be safe. If you removed it you might have less things in the cache, would would cause you to make more API calls, meaning a potential rate limit.